modules/flowable-common-rest/src/test/java/org/flowable/common/rest/api/DateFormattingTest.java

```diff
@@ -0,0 +1,54 @@
+package org.flowable.common.rest.api;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.Instant;
+import java.util.Date;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DateFormattingTest {
+
+    @Test
+    public void testValidDates() {
+        assertEquals(
```

Comment:
I believe the project standards are to use `assertThat()` form instead of `assertEquals()`